### PR TITLE
Route issue: the info won't pass to App.js to show on edit page.

### PR DESCRIPTION
**Pull request by hawk198723**

### CODE REVIEW

File: app/controllers/auction_items_controller.rb [hunk 1]
```diff
@@ -6,16 +6,33 @@ def index
  end
 
   def create
-    auction_item = AuctionItem.create(auction_item_params)
+    binding.pry
```
Comment (sjproctor) [inline app/controllers/auction_items_controller.rb]: Don't leave your debugger tools in here!

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -28,9 +29,24 @@ class App extends Component {
   };
 
   createAuction = (auction) => {
-    fetch("/auction_items", { method: "POST", body: auction });
+    fetch("/auction_items", { method: "POST",headers:{'Content-Type': 'application/json'}, body: JSON.stringify({auction_item:auction}) });
```
Comment (sjproctor) [inline app/javascript/components/App.js]: This needs some indentation and line spacing love.

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -28,9 +29,24 @@ class App extends Component {
...
+			},
+			method: 'PATCH'
+		}).then((response) =>
+			response
+				.json()
+				.then((payload) => this.readAuction())
+				.catch((errors) => console.log('Auction create errors:', errors))
+		);
+	};
+
```
Comment (sjproctor) [inline app/javascript/components/App.js:55]: This indentation needs some cleanup.

File: app/javascript/components/App.js [hunk 4]
```diff
@@ -28,9 +29,24 @@ class App extends Component {
   };
 
   createAuction = (auction) => {
-    fetch("/auction_items", { method: "POST", body: auction });
+    fetch("/auction_items", { method: "POST",headers:{'Content-Type': 'application/json'}, body: JSON.stringify({auction_item:auction}) });
   };
 
+  updateAuction = (updateAuction, id) => {
+		fetch('http://localhost:3000/auction_items/${id}', {
```
Comment (sjproctor) [inline app/javascript/components/App.js]: You don't need to fetch to localhost. You can just fetch to 'auction_items/${id}'. Also, this doesn't look like string interpolation syntax.

File: app/javascript/components/pages/AuctionEdit.js [hunk 5]
```diff
@@ -0,0 +1,125 @@
+import React, { Component } from "react";
+import { Form, FormGroup, Label, Input, Button } from "reactstrap";
+import { Redirect } from "react-router-dom";
+
+class AuctionEdit extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props.auction);
```
Comment (sjproctor) [inline app/javascript/components/pages/AuctionEdit.js]: Make sure you remove console.logs!

**Review by sjproctor — CHANGES_REQUESTED** [review]
Couple things to clean up.

File: app/javascript/components/App.js (hunk 2 above)
Comment (zan-rosenthal) [reply app/javascript/components/App.js]: @hawk198723 If you install this extension and enable format on save in the VSCode settings, it should take care of the spacing and indentation automatically:  https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: app/javascript/components/App.js (hunk 4 above)
Comment (zan-rosenthal) [reply app/javascript/components/App.js]: I could be wrong, but I think you will need a leading forward slash to indicate its a request to the same domain, ie `/auction_items/${id}`

File: app/javascript/components/App.js (hunk 2 above)
Comment (hawk198723) [reply app/javascript/components/App.js]: thank you, Zan, I already had it, I think I get some code from ATOM and cause the issue, I already fix the indentation.

File: app/javascript/components/App.js [hunk 6]
```diff
@@ -28,7 +29,22 @@ class App extends Component {
   };
 
   createAuction = (auction) => {
-    fetch("/auction_items", { method: "POST", body: auction });
+    fetch("/auction_items", { method: "POST", headers:{ 'Content-Type': 'application/json' }, body: JSON.stringify({auction_item:auction}) });
```
Comment (sjproctor) [inline app/javascript/components/App.js]: Still need some love here ^^

**Review by sjproctor — CHANGES_REQUESTED** [review]
Bit more clean up on your fetch request!

**Review by sjproctor — APPROVED** [review]
Beautiful!

**Pull request merged**